cristin-person/src/test/java/no/unit/nva/cristin/person/handler/userAdministrator.feature

```diff
@@ -0,0 +1,59 @@
+Feature: API tests for Cristin Person fetch
+
+  Background:
+    * def domainName = java.lang.System.getenv('DOMAIN_NAME')
+    * def basePath = java.lang.System.getenv('BASE_PATH')
+    * def CRISTIN_BASE =  'https://' + domainName +'/' + basePath
+    * def username = java.lang.System.getenv('TESTUSER_FEIDE_ID')
+    * def password = java.lang.System.getenv('TESTUSER_PASSWORD')
+    * def simple_user_name = java.lang.System.getenv('SIMPLE_TESTUSER_FEIDE_ID')
+    * def simple_user_password = java.lang.System.getenv('SIMPLE_TESTUSER_PASSWORD')
+    * def cognitoClientAppId = java.lang.System.getenv('COGNITO_CLIENT_APP_ID')
+    * def cognitoUserpoolId = java.lang.System.getenv('COGNITO_USER_POOL_ID')
+    * def tokenGenerator = Java.type('no.unit.nva.cognito.CognitoUtil')
+    * def token = tokenGenerator.loginUser(username, password, cognitoUserpoolId, cognitoClientAppId)
+    * def simpleUserToken = tokenGenerator.loginUser(simple_user_name, simple_user_password, cognitoUserpoolId, cognitoClientAppId)
+    * def samplePersonIdentifier = '515114'
+    * def PROBLEM_JSON_MEDIA_TYPE = 'application/problem+json'
+    Given url CRISTIN_BASE
+    * print 'Current base url: ' + CRISTIN_BASE
+
+  Scenario: Fetch returns classified person data when authenticated and authorized
+    Given path '/person/' + samplePersonIdentifier
+    And header Authorization = 'Bearer ' + token
+    And request
```

Comment:
empty request? No need to fix i dont think its a problem but maybe its more for post etc